app/_hub/kong-inc/session/_index.md

```diff
@@ -39,24 +39,99 @@ params:
+    - name: remember_rolling_timeout
+      minimum_version: "3.2.x"
+      required: false
+      default: 604800
+      datatype: integer
+      description: Persistent session rolling timeout in seconds.
+    - name: remember_absolute_timeout
+      minimum_version: "3.2.x"
+      required: false
+      default: 2592000
+      datatype: integer
+      description: Persistent session absolute timeout in seconds.
+    - name: request_headers
+      minimum_version: "3.2.x"
+      required: false
+      default: null
+      datatype: array of string elements
+      description: |
+        List of information to include (as headers) in the request to upstream. 
+        Accepted values are: `id`, `audience`, `subject`, `timeout`, `idling-timeout`, `rolling-timeout`, and
+        `absolute-timeout`.
+        For example, { "id", "timeout" } will set both `Session-Id` and `Session-Timeout` in the request headers.
+    - name: response_headers
+      minimum_version: "3.2.x"
+      required: false
+      default: null
+      datatype: array of string elements
+      description: |
+        List of information to include (as headers) in the response to downstream. 
+        Accepted values are: `id`, `audience`, `subject`, `timeout`, `idling-timeout`, `rolling-timeout`, and
+        `absolute-timeout`.
+        For example: { "id", "timeout" } will inject both `Session-Id` and `Session-Timeout` in the response headers.
     - name: cookie_name
       required: false
       default: '`session`'
       datatype: string
       description: The name of the cookie.
     - name: cookie_lifetime
+      maximum_version: "3.1.x"
       required: false
       default: 3600
       datatype: number
       description: The duration in seconds that the session will remain open.
+    - name: rolling_timeout
+      minimum_version: "3.2.x"
+      required: false
+      default: 3600
+      datatype: integer
+      description: |
+        The session cookie rolling timeout in seconds.
+        Specifies how long the session can be used until it needs to be renewed.
+    - name: absolute_timeout
+      minimum_version: "3.2.x"
+      required: false
+      default: 86400
+      datatype: integer
+      description: |
+        The session cookie absolute timeout in seconds.
+        Specifies how long the session can be used until it is no longer valid.
     - name: cookie_idletime
+      maximum_version: "3.1.x"
       required: false
       datatype: number
       description: |
         The cookie idle time (in seconds); if a cookie is not used for this time
         period, the session becomes invalid. This value is not set by default,
         meaning idle time checks are disabled.
+    - name: idling_timeout
+      minimum_version: "3.2.x"
+      required: false
+      default: 900
+      datatype: integer
+      description: The session cookie idle time in seconds.
     - name: cookie_renew
+      maximum_version: "3.1.x"
```

Comment:
Is this accurate? The changelog entry says the parameter is deprecated, not removed. If it's only deprecated, we should leave it in 3.2.x and add a note to the description saying that it's deprecated, planned to be removed, and to please use X parameter instead.

Reply:
`Removed` is probably a better fit, I fixed that. The thing is, the field is still accepted without producing errors. if configured, but it is ineffective. `Deprecated` probably suggests it still works though, so that was not good.